api/src/test/java/org/apache/iceberg/io/TestCloseableIterable.java

```diff
@@ -107,9 +108,9 @@ public void testWithCompletionRunnable() throws IOException {
     try (CloseableIterable<Integer> iter =
         CloseableIterable.whenComplete(
             CloseableIterable.combine(items, () -> {}), completionCounter::incrementAndGet)) {
-      iter.forEach(val -> Assertions.assertThat(completionCounter.get()).isEqualTo(0));
+      iter.forEach(val -> assertThat(completionCounter.get()).isZero());
     }
-    Assertions.assertThat(completionCounter.get()).isEqualTo(1);
+    assertThat(completionCounter.get()).isOne();
```

Comment:
is there any value in changing this from `isEqualTo()` to `isOne()`?

Reply:
Thank you Eduard for taking time to review this. Would keep this in mind.
But one qq: Since you have already spent time in reviewing these changes should i revert the isEqualTo() to isOne()/isZero() changes or keep them as it is for now ?

Reply:
I'm ok leaving them now, but just something to consider for future PRs